main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+## dIBC
+
+Dynamic version of the [Inter-Blockchain Communication](#ibc) protocol. 
+See [here](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/networking.md) for more details.
+
+## E()
+
+(Also referred to as *eventual send*) `E()` is a local "bridge" function that invokes methods on remote objects, for example
+in another vat, machine, or blockchain. It takes a local representative (a proxy) for a remote object as an argument and 
+sends messages to it using normal message-sending syntax. The local proxy forwards all messages to the remote object to deal with. 
+All `E()` calls return a promise for the eventual returned value. For more detail, see 
+the [`E()` section in the Distributed JavaScript page](/distributed-programming.md#communicating-with-remote-objects-using-e).
 
 ## ERTP
-*Electronic Rights Transfer Protocol* is a uniform way of transferring tokens and other digital assets, both [fungible](#fungible) and [non-fungible](#non-fungible), in JavaScript. All kinds of digital assets can easily be created and they can be all be transferred in exactly the same ways, with exactly the same security properties.
+
+*Electronic Rights Transfer Protocol* is a uniform way of transferring tokens and other digital assets, 
+both [fungible](#fungible) and [non-fungible](#non-fungible), in JavaScript. All kinds of digital assets
+can easily be created and they can be all be transferred in exactly the same ways, with exactly the same security properties.
 
 It uses [object capabilities](#object-capabilities) to enforce access control. Instead of having 
 to prove ownership of a corresponding private key, if your program has a 
 reference to an object, it can call methods on that object. If it doesn't 
 have a reference, it can't. For more on object capabilities, see [this post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
 
-Key ERTP concepts include [Issuers](#issuer), [mints](#mint), 
-[Purses](#purse), [Payments](#payment), [Brands](#brand), and [Amounts](#amounts). Also 
+Key ERTP concepts include [Issuers](#issuer), [Mints](#mint), 
+[Purses](#purse), [Payments](#payment), [Brands](#brand), and [Amounts](#amount). Also 
 see the [ERTP Introduction](/getting-started/ertp-introduction.md),
 [ERTP Guide](/ertp/guide/), and [ERTP API](/ertp/api/).
 
+## Escrow
+
+To give assets for a possible transaction to an impartial third party, who keeps them until specified conditions are satisfied. 
+For example, Alice wants to sell Bob a ticket for $100. Alice escrows the ticket, and Bob escrows the $100, with Carol. Carol
+does not give Alice the $100 or Bob the ticket until she has both items. Since neither Alice nor Bob ever holds both items at
+once, they don't have to trust each other to do the transaction. Zoe automatically escrows payments for transaction offers.
+
```

Comment:
I don't think we should define escrow in a general way with a untrusted third-party such as Carol, when users should use Zoe for escrow.